debian/ooniprobe-cli.install

```diff
@@ -0,0 +1 @@
+dist/linux/amd64/ooniprobe usr/sbin
```

Comment:
Why `sbin`? It strikes me more as a tool for `bin`

Reply:
Yeah I would also have expected that too.